india_compliance/public/js/setup_wizard.js

```diff
@@ -65,6 +104,15 @@ function update_erpnext_slides_settings() {
 async function autofill_company_info(slide) {
     const gstin = slide.get_input("company_gstin").val();
     const gstin_field = slide.get_field("company_gstin");
+
+    set_gstin_description(gstin_field);
```

Comment:
We don't want this in the setup wizard.
It should be enough to have a red-bordered input box, with small text in red saying invalid GSTIN (only if it's invalid)

Reply:
Used `field_obj.$input.css("border", "1px solid red");`